backend/tests/unit/simulation/alarm_limits.txt

```diff
@@ -0,0 +1,34 @@
+Scenario: Alarm Limit services behave properly
+Given: A PCAC Alarm Limit service
+When: parameter service is not equal to pc_ac or HFNC
+Then: Alarm Limits are not updated as per Alarm Limit requests
+
+Scenario: Alarm Limit services behave properly
+Given: A PCAC Alarm Limit service
+When: parameter service is equal to pc_ac
+Then: Alarm Limits are updated as per Alarm Limit requests
+
+Scenario: Alarm Limit services behave properly
+Given: A PCAC Alarm Limit service
+When: Requested alarm limit lower value is lesser than defined lower value
+Then: Alarm Limits are not updated as per Alarm Limit requests
+
+Scenario: Alarm Limit services behave properly
+Given: A PCAC Alarm Limit service
+When: Requested alarm limit upper value is greater than defined upper value
```

Comment:
Instead of "the defined upper value", I think it would be clearer to say "the maximum allowed upper value for the alarm limit". This comment also applies to other tests in this file.
```suggestion
When: The requested upper value for the SpO2 alarm limits range is greater than the maximum allowed upper value for the range.
```

Reply:
updated as suggested.